core/domain/suggestion_services_test.py

```diff
@@ -1643,15 +1643,1249 @@ def test_get_voiceover_application_from_model_with_invalid_type_raise_error(
+
+        reviewable_suggestion_email_infos = (
+            suggestion_services
+            .get_suggestions_waiting_for_review_info_to_notify_reviewers(
+                [self.reviewer_1_id, self.reviewer_2_id]
+            )
+        )
+
+        self.assertEqual(len(reviewable_suggestion_email_infos), 2)
+        self._assert_reviewable_suggestion_email_infos_are_in_correct_order(
+            reviewable_suggestion_email_infos[0],
+            expected_reviewable_suggestion_email_infos_reviewer_1)
+        self._assert_reviewable_suggestion_email_infos_are_in_correct_order(
+            reviewable_suggestion_email_infos[1],
+            expected_reviewable_suggestion_email_infos_reviewer_2)
+
+    def test_get_returns_infos_for_reviewer_with_multi_rights_past_limit(
+            self):
+        user_services.allow_user_to_review_translation_in_language(
+            self.reviewer_1_id, 'hi')
+        user_services.allow_user_to_review_question(self.reviewer_1_id)
+        translation_suggestion_1 = (
+            self._create_translation_suggestion_with_language_code_and_author(
+                'hi', self.author_id))
+        # Create additional suggestions so that we pass the
+        # MAX_NUMBER_OF_SUGGESTIONS_PER_REVIEWER limit regardless of suggestion
+        # type.
+        self._create_question_suggestion_with_skill_id_and_author_id(
+            'skill_1', self.author_id)
+        self._create_translation_suggestion_with_language_code_and_author(
+            'hi', self.author_id)
+        self._create_question_suggestion_with_skill_id_and_author_id(
+            'skill_1', self.author_id)
+        expected_reviewable_suggestion_email_infos = (
+            self._create_reviewable_suggestion_email_infos_from_suggestions(
+                [translation_suggestion_1]))
+
+        with self.swap(
+            suggestion_services, 'MAX_NUMBER_OF_SUGGESTIONS_PER_REVIEWER', 1):
+            reviewable_suggestion_email_infos = (
+                suggestion_services
+                .get_suggestions_waiting_for_review_info_to_notify_reviewers(
+                    [self.reviewer_1_id]))
+
+        self.assertEqual(len(reviewable_suggestion_email_infos), 1)
+        self._assert_reviewable_suggestion_email_infos_are_in_correct_order(
+            reviewable_suggestion_email_infos[0],
+            expected_reviewable_suggestion_email_infos)
+
+
 class CommunityContributionStatsUnitTests(test_utils.GenericTestBase):
     """Test the functionality related to updating the community contribution
     stats.
 
-    Note: testing whether resubmiting a rejected translation increases the
-    translation suggestion count could not be tested because the
-    pre_update_validate method in suggestion_registry.py is not implemented for
-    translation suggestions. As a result, rejected translation suggestions
-    cannot be resubmitted for review.
+    TODO(kaylahardie): It is currently not possible to resubmit a rejected
```

Comment:
Prefer the format: ``TODO(#1234)``. It's better to use an issue to track a TODO than an author since TODOs often stick around for a really long time, and it encourages anybody to fix it versus assuming that the author needs to.

Reply:
Done!